src/environments.js

```diff
@@ -66,10 +66,12 @@ var environmentDefinitions = [
             var parser = this;
             // Currently only supports alignment, no separators like | yet.
             colalign = colalign.value.map ? colalign.value : [colalign];
-            colalign = colalign.map(function(node) {
+            var cols = colalign.map(function(node) {
                 var ca = node.value;
                 if ("lcr".indexOf(ca) !== -1) {
-                    return ca;
+                    return {
+                        align: ca
+                    };
```

Comment:
I've thought about similar solutions myself. I think `value` is over-used in the codebase already, so I'd have `{type:"content", align:"c", index:0}` and `{type:"space", width:quad}` and so on. I just started on this and realized that if I want to avoid excessive code duplication, this will require some major work, so it would delay the `{cases}` environment considerably. Shall we postpone this?